cmd/kubeadm/app/util/marshal.go

```diff
@@ -33,6 +33,8 @@ import (
 	"k8s.io/kubernetes/cmd/kubeadm/app/constants"
 )
 
+const mediaType = runtime.ContentTypeYAML
```

Comment:
Maybe name it `yamlMediaType` since the references won't be in the immediate function anymore and is therefore less obvious?